dts/bindings/gpio/nuvoton,nct38xx-gpio-port.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2021 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Nuvoton NCT38XX series I2C-based GPIO expander port
+
+compatible: "nuvoton,nct38xx-gpio-port"
+
+include: [gpio-controller.yaml, base.yaml]
+
+properties:
+    label:
+      required: true
+
+    reg:
+      required: true
+
+    pin_mask:
+      type: int
+      required: false
+      default: 0xff
+      description: |
+        Not every GPIOs are usable for different NCT38XX series. This property
+        indicates the usable GPIO mask.
+
+    pinmux_mask:
+      type: int
+      required: false
+      default: 0xff
+      description: |
+        NCT38XX series port 0 has Pin Multiplexing functionality. However, not
+        every GPIOs have pinmux controller functionality. This property
+        indicates the usable GPIO pinmux mask.
```

Comment:
All uses of `default` must be justified in the description. Please make sure to read https://docs.zephyrproject.org/latest/guides/dts/bindings.html#rules-for-mainline-bindings if you have not already done so.

Reply:
Thanks for sharing this. This might not safely default. I removed the default here.